src/pages/Register/Register.jsx

```diff
@@ -0,0 +1,113 @@
+
+import RegisterSchema from "./RegisterSchema";
+
+const Register = () => {
+  const formik = useFormik({
+    initialValues: {
+      name: "",
+      email: "",
+      password: "",
+    },
+    initialErrors: {
+      name: false,
+      email: false,
+      password: false,
+    },
+    initialTouched: {
+      name: false,
+      email: false,
+      password: false,
+    },
+    validationSchema: RegisterSchema,
+    handleSubmit(values) {
+      console.log(values);
+    },
+  });
+  return (
+    <Container className="outer-container">
+      <form className="inner-container" onSubmit={formik.handleSubmit}>
+        <Grid container alignItems="center" direction="row" justify="center">
+          <Grid item className="title-item">
+            <Typography variant="h1">Bienvenide a Healthy Dev</Typography>
+          </Grid>
+          <Grid item className="name-item">
+            <TextField
+              fullWidth
+              required
+              aria-describedby="Nombre"
+              error={formik.touched.name && formik.errors.name}
+              id="name"
+              name="name"
+              placeholder="Nombre"
+              value={formik.values.name}
+              variant="filled"
+              onBlur={formik.handleBlur}
+              onChange={formik.handleChange}
+            />
+            <FormHelperText error={formik.touched.name && formik.errors.name} variant="filled">
+              {formik.touched.name && formik.errors.name}
+            </FormHelperText>
+          </Grid>
+          <Grid item className="email-item">
+            <TextField
+              fullWidth
+              required
+              aria-describedby="Email"
+              error={formik.touched.email && formik.errors.email}
+              id="email"
+              name="email"
+              placeholder="Email"
+              value={formik.values.email}
+              variant="filled"
+              onBlur={formik.handleBlur}
+              onChange={formik.handleChange}
+            />
+            <FormHelperText error={formik.touched.email && formik.errors.email} variant="filled">
+              {formik.touched.email && formik.errors.email}
+            </FormHelperText>
+          </Grid>
+          <Grid item className="password-item">
+            <TextField
+              fullWidth
+              required
+              aria-describedby="Crear Password"
+              error={formik.touched.password && formik.errors.password}
+              id="password"
+              name="password"
+              placeholder="Crear Password"
+              type="password"
+              value={formik.values.password}
+              variant="filled"
+              onBlur={formik.handleBlur}
+              onChange={formik.handleChange}
+            />
+            <FormHelperText error={formik.touched.password && formik.errors.password} variant="filled">
```

Comment:
Intercambiar valores para evitar error de tipos

```suggestion
            <FormHelperText error={formik.errors.password && formik.touched.password} variant="filled">
```